### DISCUSSION TITLE
Flere beskeder

### DISCUSSION THREAD

**Discussion by runeod** [scaarup/aula#135] [General]
Hej Søren.

Er det muligt at se alle ulæste beskeder og ikke kun den seneste?
Er det muligt, at markere den som læst?

På forhånd tak for svar :-)

//Rune

**Comment by scaarup** [comment]
Hej Rune.

Jo, det kunne være en mulighed at få vist op til X antal ulæste beskeder - tror jeg godt vi kan lave. Men at markere dem som læst, bliver lidt sværere. Man kunne evt. lave en service, der markerede alle ulæste som læste.
